rpc-server/src/modules/transactions/methods.rs

```diff
@@ -93,6 +101,7 @@ pub async fn tx_status(
             near_jsonrpc_client::methods::EXPERIMENTAL_tx_status::RpcTransactionStatusRequest {
                 transaction_info: tx_status_request.transaction_info,
             },
+            response_state,
```

Comment:
I cannot comment on line 110 because it's not a part of the changes you make, but I've noticed there is a target `shadow-data-consistency` while on 113 the target is `shadow_data_consistency` (note the delimiters), please could you check while you're here to fix them?